src/Tizen.NUI.Components/Utils/StyleManager.cs

```diff
@@ -160,16 +151,7 @@ public ViewStyle GetViewStyle(string style)
                 return null;
             }
 
-            if (themeStyleSet.ContainsKey(style) && themeStyleSet[style].ContainsKey(Theme))
-            {
-                return (themeStyleSet[style][Theme])?.GetViewStyle();
-            }
-            else if (defaultStyleSet.ContainsKey(style))
-            {
-                return (defaultStyleSet[style])?.GetViewStyle();
-            }
-
-            return null;
+            return (ThemeManager.CurrentTheme.GetStyle(style) ?? ThemeManager.DefaultTheme.GetStyle(style))?.Clone();
```

Comment:
on mobile or tv profile, ThemeManager.CurrentTheme is null.
a null check might be added.